### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## PR Summary

@ngoldbaum asked me to do this.

Composed using https://github.com/jsoref/spelling

## PR Checklist

- [ ] Code passes flake8 checker

### CODE REVIEW

**Comment by ngoldbaum** [comment]
Awesome! Thanks for this. 

I think we’ll likely want this as one commit before it gets merged. 

I will try to look this over in detail this week and let you know if any of these count as API changes.

**Comment by jsoref** [comment]
My initial series had at least one error, it's much easier to review/fix such things before squashing.

File: doc/source/quickstart/3)_Simple_Visualization.ipynb [hunk 1]
```diff
@@ -64,7 +64,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `show` command simply sends the plot to the IPython notebook.  You can also call `p.save()` which will save the plot to the file system.  This function accepts an argument, which will be pre-prended to the filename and can be used to name it based on the width or to supply a location.\n",
+    "The `show` command simply sends the plot to the IPython notebook.  You can also call `p.save()` which will save the plot to the file system.  This function accepts an argument, which will be pre-prepended to the filename and can be used to name it based on the width or to supply a location.\n",
```
Comment (brittonsmith) [inline doc/source/quickstart/3)_Simple_Visualization.ipynb]: I think this should just be "prepended".

Comment (jsoref) [reply doc/source/quickstart/3)_Simple_Visualization.ipynb]: oops, yes, sorry. the spell checker didn't like `prended`, I knew the word was supposed to be prepended, but forgot to do that when I let my browser suggest the replacement.

**Pull request merged**